### PR TITLE
new formula: libsoundio 1.0.3

### PR DESCRIPTION
**Pull request by LinusU**
This is an awesome sound library. From their website:

> libsoundio is a lightweight abstraction over various sound drivers. It provides a well-documented API that operates consistently regardless of the sound driver it connects to. It performs no buffering or processing on your behalf; instead exposing the raw power of the underlying backend.

> libsoundio is appropriate for games, music players, digital audio workstations, and various utilities.

> libsoundio is serious about robustness. It even handles out of memory conditions correctly.

### CODE REVIEW

File: Library/Formula/libsoundio.rb [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        if (!soundio) { return 1; }
+        if (soundio_connect(soundio)) return 1;
+
+        soundio_flush_events(soundio);
+        soundio_destroy(soundio);
+
+        return 0;
+      }
+    EOS
+    system ENV.cc, "-L/usr/local/lib", "-lsoundio", "test.c", "-o", "test"
```
Comment (DomT4) [inline Library/Formula/libsoundio.rb]: Please point this to `#{lib}` instead of the top-level prefix.

File: Library/Formula/libsoundio.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+class Libsoundio < Formula
+  desc "cross-platform audio input and output"
```
Comment (DomT4) [inline Library/Formula/libsoundio.rb]: Please capitalise the start of the `desc`.

**Comment by LinusU** [comment]
Thanks for the comments, I've updated accordingly :+1:

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

For future reference the preferred commit message format for simple version updates is `libsoundio 1.0.3 (new formula)`.

**Comment by LinusU** [comment]
Thanks, I'll nail the commit message next time :)

**Pull request closed without merging**